swupd/packs.go

```diff
@@ -113,6 +113,26 @@ func CreateAllDeltas(outputDir string, fromVersion, toVersion, numWorkers int) e
 	return nil
 }
 
+func writeDeltaManifest(tw *tar.Writer, outputDir string, dManifest *Manifest, toVersion uint32) error {
+	if dManifest == nil || len(dManifest.Files) == 0 {
+		return nil
+	}
+
+	dManifest.Header.FileCount = uint32(len(dManifest.Files))
+	dManifestPath := filepath.Join(outputDir, fmt.Sprint(toVersion),
+		fmt.Sprintf("Manifest.%s.D.%d", dManifest.Name, dManifest.Header.Version))
+
+	if err := dManifest.WriteManifestFile(dManifestPath); err != nil {
```

Comment:
hmm, we should have made manifests an interface when we started having different types of them... not necessary for this PR, but it could be updated later.